src/azure-cli/azure/cli/command_modules/synapse/tests/latest/test_synapse_scenario.py

```diff
@@ -346,6 +347,57 @@ def test_sql_pool_audit_policy(self):
                      self.check('state', 'Enabled')
                  ])
 
+    @record_only()
```

Comment:
why record only here?

Reply:
The related test need to create workspace, create sql pool, create storage and assign permissions, it is hard to do in live mode.